package.json

```diff
@@ -55,13 +56,17 @@
     "stream-browserify": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.17.9",
```

Comment:
can we use `ts-jest` vs introducing babel?

Reply:
why? babel is a far superior transpiler to tsc, and should always be used exclusively to transpile, including for TS.

Reply:
If thats the case, should we upgrade the build to use babel vs tsc?

Reply:
1000% yes

Reply:
ty both! will make that update 👌